Extensions/PlatformBehavior/Extension.cpp

```diff
@@ -501,6 +514,81 @@ void DeclarePlatformBehaviorExtension(gd::PlatformExtension& extension) {
         .SetFunctionName("IgnoreDefaultControls")
         .SetIncludeFile("PlatformBehavior/PlatformerObjectRuntimeBehavior.h");
 
+    aut.AddAction("CanGrabPlatforms",
+                  _("Can Grab Platform"),
+                  _("De/activate when the platformer object can grab the platform."),
```

Comment:
Enable (or disable) the ability of the object to grab platforms when falling near to one.